src/Sinch/Fax/Faxes/FaxesClient.cs

```diff
@@ -0,0 +1,196 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Net.Http;
+using System.Runtime.CompilerServices;
+using System.Threading;
+using System.Threading.Tasks;
+using Sinch.Core;
+using Sinch.Logger;
+
+namespace Sinch.Fax.Faxes
+{
+    /// <summary>
+    ///     The Fax API allows you to send and receive faxes.
+    ///     You can send faxes to a single recipient or to multiple recipients.
+    ///     You can also receive faxes and download them.
+    /// </summary>
+    public interface ISinchFaxFaxes
+    {
+        /// <summary>
+        ///     Create and send a fax.<br/><br/>
+        ///     Fax content may be supplied via one or more files or URLs of supported filetypes.<br/><br/>
+        ///     If you supply a callbackUrl the callback will be sent as multipart/form-data with the content
+        ///     of the fax as an attachment to the body, unless you specify callbackUrlContentType as application/json.
+        ///     If you specify more than one receiver, list will have more then one faxes.
+        /// </summary>
+        /// <param name="request"></param>
+        /// <param name="cancellationToken"></param>
+        /// <returns></returns>
+        public Task<List<Fax>> Send(SendFaxRequest request, CancellationToken cancellationToken = default);
```

Comment:
I see you have decided to enforce the usage of an array of faxes for the response. For the request part, this is similar to what is proposed for SMS batches, but for the response, it may be an additional burden for the SDK user to have to manage a list of faxes when there is a single recipient.

Reply:
Okay, I can play around it